pkg/kubelet/pleg/generic.go

```diff
@@ -366,7 +366,7 @@ func (g *GenericPLEG) getPodIPs(pid types.UID, status *kubecontainer.PodStatus)
 	}
 
 	if len(status.SandboxStatuses) == 0 {
-		// Without sandboxes (which built-in runtimes like rkt don't report)
+		// If sandboxes are not present, then
```

Comment:
As a note for reviewers... I'm assuming that we still want to fallback to checking `ContainerStatuses` when no `SandboxStatuses` are present - is that a safe assumption?

Another option would be to remove this `if` statement entirely if we could be confident that `len(status.SandboxStatuses)` would ALWAYS not equal 0 when using the `cri`. I wasn't sure if that assumption was safe.

Reply:
cc @Random-Liu @yujuhong

Reply:
Since rkt is no longer a built-in runtime, and docker(shim) uses CRI, I think it's safe to remove the block entirely.

Reply:
:+1: will update :)